### ISSUE TITLE
Inline code in source code mode is using comment css instead of code css

### ISSUE THREAD

**Issue by vr8hub** [typora/typora-issues#1362]
Typora .9.9.13.6 on OSX 10.12.6.

When in source code mode, e.g. \`something like this\`, Typora is using the .cm-comment CSS instead of the .cm-code CSS.

Here is an excerpt of the two lines I have in my theme (the variables are defined at the top of the theme):
```css
.cm-s-typora-default .cm-code { color: var(--sol-magenta); }
.cm-s-typora-default .cm-comment { color: var(--sol-green); font-style:italic; }
```

Inline code in source code mode is displaying in green rather in magenta. I confirmed that this was the issue by changing the color of .cm-comment to red; the inline code then started displaying in red.

**Comment by abnerlee** [comment]
This is a upstream issue of CodeMirror, which typora use for syntax highlighting

you may try in https://codemirror.net/mode/markdown/

And therefore, this should be fixed by codemirror, and Typora will follow

**Comment by vr8hub** [comment]
I'm not using CodeMirror. I'm using Typora. *You're* using CodeMirror. I gave you the relevant CSS lines and indicated how I confirmed what the problem was. There's no way for me to pursue a problem with CodeMirror, because I'm not using CodeMirror. If you have a problem with a third-party library in your program, that's on you to fix, not me. Ignore it if you want. I have reported the problem on the program where the problem occurred.
